src/views/NotificationButton.tsx

```diff
@@ -124,12 +124,28 @@ class NotificationButton extends ComponentEx<IProps, IComponentState> {
       });
     }
 
+    const isRunningBgProcess = 
+      notifications
+      .filter((x) => x.type === 'activity' && x.progress === undefined)
+      .length >= 1 && combinedProgress.length == 0;
```

Comment:
The way this is wrapped is super hard to read.
Also, it's needlessly complicated. If "combinedProgress.length" is 0 then you already know there is no activity notification with a progress. all the filter has to look for is type === activity. And then I would advice putting the check for combinedProgress.length first, that's the cheaper (performance-wise) condition to check.
Not very relevant here but a good habit to get into.